src/style.css

```diff
@@ -1,3 +1,20 @@
-body {
+/* body {
   background-color: red;
+} */
+
+/* Add some style for the cards */
+.card {
+  width: 300px;
+  border: 1px solid black;
+  margin: 10px;
+}
+
+.img-container {
+  height: 200px;
+  overflow: hidden;
+}
+
+img {
+  width: 100%;
+  object-fit: cover;
 }
```

Comment:
The project will look even better with some more styles added to it.

Reply:
- Will add more styles after implementing all app functionality